core/kysely/scripts/migrate-hierarchy.ts

```diff
@@ -8,7 +8,14 @@ import { expect } from "utils";
 
 import { createLastModifiedBy } from "~/lib/lastModifiedBy";
 import { slugifyString } from "~/lib/string";
-import { db } from "../database";
+import { createDatabase } from "../database-init";
+
+const db = createDatabase({
+	url: `DATABASE_URL=postgresql://${process.env.PGUSER}:${process.env.PGPASSWORD}@${process.env.PGHOST}:${process.env.PGPORT}/${process.env.PGDATABASE}`,
+	logLevel: "debug",
+	debug: true,
+	env: "production",
```

Comment:
```suggestion
	debug: true,
```

I would just skip passing the env if you take my suggested changes above